packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -300,7 +300,9 @@ class TransitLegBody extends Component<Props, State> {
                     alertCount: alerts.length
                   }}
                 />
-                <S.CaretToggle expanded={alertsExpanded} />
+                {!shouldOnlyShowAlertsExpanded && (
```

Comment:
Wait doesn't `shouldOnlyShowAlertsExpanded` just impact how the alerts are rendered? The caret should still be there if users want to collapse them after render, right?

Reply:
That behavior doesn't work right now